server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -94,6 +95,14 @@ class DeltaSharingServiceExceptionHandler extends ExceptionHandlerFunction {
             Map(
               "errorCode" -> ErrorCode.INVALID_PARAMETER_VALUE,
               "message" -> cause.getMessage)))
+      case _: DeltaStandaloneException =>
+        HttpResponse.of(
+          HttpStatus.BAD_REQUEST,
```

Comment:
Is it safe to always map delta standalone exceptions to be bad requests?

Reply:
In this case it is a bad request instead of an internal error. If we do not expose all `DeltaStandaloneException`, then we have to do a special match for this case, which also works, but don't feel it's obviously better than the current solution.

Will quickly sync with Ryan tomorrow to get more context.

Reply:
I added a more specific check in: https://github.com/delta-io/delta-sharing/pull/150